vindaloo/vindaloo.py

```diff
@@ -239,6 +240,23 @@ def k8s_deploy(self) -> None:
                     self._out('Waiting for rollout {} to finish'.format(deployment_name))
                     self.cmd(["kubectl", "rollout", "status", "deployment", deployment_name])
 
+            for yaml_conf in self.config_module.K8S_OBJECTS.get('job', []):
+                if isinstance(yaml_conf, JsonSerializable):
+                    job_name = yaml_conf.name
+                if job_name:
+                    self._out('Waiting for job {} to finish'.format(job_name))
+
+                    succeeded, failed = None, None
+                    while not succeeded and not failed:
+                        time.sleep(60)  # one minute wait for not messing up kube
+                        succeeded = self.cmd(["kubectl", "get", "job", job_name, "-o", "jsonpath={.status.succeeded}"]).stdout
+                        failed = self.cmd(["kubectl", "get", "job", job_name, "-o", "jsonpath={.status.failed}"]).stdout
+                        if succeeded:
+                            self._out("Job {} ended successfully.".format(job_name))
+                        if failed:
+                            self._out("Job {} failed.".format(job_name))
+                            exit(1)
+
```

Comment:
Once, in very important moment, I've met situation, when the pod was in loop on failing readiness probe.